earth_enterprise/rpms/build.gradle

```diff
@@ -43,6 +43,17 @@ apply plugin: "nebula.ospackage"
 
 import org.opengee.shell.GeeCommandLine
 
+// Varables used for specifying location of two files
+// 'gee_long_version.txt' and 'gepgcommon'
+def working_dir = "../src/"
```

Comment:
This seems like a misleading name. The Gradle working directory is usually the project directory.

Reply:
- [ ] Rename variable to, e.g. `src_dir`.